src/ProfilePage/Api/profileApi.ts

```diff
@@ -12,48 +12,33 @@ export const getCommits = async (repoInfo: RepoItem): Promise<CommitItem[]> => {
     return [];
   }
 
-  try {
-    const commitsUrl = `https://api.github.com/repos/${repoInfo.full_name}/commits`;
-    const commitsData = await getPaginatedData(commitsUrl);
-
-    const newCommits = commitsData.map((commit: CommitItem) => ({
-      sha: commit.sha,
-      commit: commit.commit,
-      parents: commit.parents,
-    }));
-
-    return newCommits;
-  } catch (error) {
-    console.error("Error fetching commits:", error);
-    return [];
-  }
+  const commitsUrl = `https://api.github.com/repos/${repoInfo.full_name}/commits`;
```

Comment:
maybe consider having a baseUrl set to ` https://api.github.com/repos/${repoInfo.full_name}/ ` then just make reference to that in your commitsUrl and branchesUrl